atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -81,5 +81,6 @@
     public static final boolean DEFAULT_ENABLE_SWEEP = false;
     public static final long DEFAULT_SWEEP_PAUSE_MILLIS = 5 * 1000;
     public static final int DEFAULT_SWEEP_BATCH_SIZE = 1000;
+    public static final int DEFAULT_SWEEP_CELL_BATCH_SIZE = 100_000_000;
```

Comment:
how did we arrive at this 100MM cell default batch size? how much memory does this require?